ext/XS-APItest/APItest.xs

```diff
@@ -3947,7 +3947,7 @@ CODE:
         av_push(av, SvREFCNT_inc(TOPs));
         break;
 
-    case G_ARRAY:
+    case G_LIST:
```

Comment:
I kinda think APItest.xs should use the old stuff since it's testing the API and the API is what we expect non-core modules to use (and possibly steal from APItest.xs's examples)